best-practices/add_network_policy/kuttl-test/01-policy.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+apply:
+- ../add_network_policy.yaml
+assert:
+- policy-assert.yaml
+unitTest: false
```

Comment:
Empty last line is recommended, though not urgent. This red mark above in a few other files too. (https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log).